build-cluster-ec2.yml

```diff
@@ -89,19 +89,20 @@
 
 - name: Setup Lightbend
   hosts: nodes
+  vars_files:
+    - "{{ VARS_FILE }}"
```

Comment:
If placed after `user: "{{ REMOTE_USER }}"`, `REMOTE_USER` var is not resolved. I don't know why.